lib/web/apiserver_test.go

```diff
@@ -2481,6 +2483,126 @@ func TestCheckAccessToRegisteredResource(t *testing.T) {
 	}
 }
 
+func TestAuthExport(t *testing.T) {
+	ctx := context.Background()
+
+	env := newWebPack(t, 1)
+	clusterName := env.server.ClusterName()
+
+	proxy := env.proxies[0]
+	pack := proxy.authPack(t, "test-user@example.com", nil)
+
+	validateTLSCertificateDERFunc := func(t *testing.T, b []byte) {
+		cert, err := x509.ParseCertificate(b)
+		require.NoError(t, err)
+		require.NotNil(t, cert)
+		require.Equal(t, cert.Subject.CommonName, "localhost", "unexpected certificate subject CN")
```

Comment:
nit: _want_ before _got_ for testify assertions.

```suggestion
		require.Equal(t, "localhost", cert.Subject.CommonName, "unexpected certificate subject CN")
```